### PR TITLE
Write JS for Search in Page Type Food

### LINKED ISSUE #43: [Js] search function (typeFood page)

### PR DESCRIPTION
**Pull request by muhannaAtalla20**
Relates #43

### CODE REVIEW

**Review by IsraaHamdi — CHANGES_REQUESTED** [review]
edit the title and when click the categories in categoriesFood page that all return just Beef  so edit this also

**Comment by muhannaAtalla20** [comment]
> edit the title and when click the categories in categoriesFood page that all return just Beef so edit this also

This one has a issues of its own

File: js/typeFood.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const holder = document.querySelector(".cards-food");
+/* End Selectors */
+/* Start Event Listener */  
+let final = [];
+searchInp.addEventListener("keyup", (e) => {
+    const searchString = e.target.value;
+    let filterd = final.meals.filter((ch) => {
+        return ch.strMeal.toUpperCase().includes(searchString.toUpperCase());
+    });
+    console.log(e.key)
```
Comment (yaqootturman) [inline js/typeFood.js]: delete

File: js/typeFood.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+/* End Event Listener */
+/* Start Function */ 
+const loadCh = async () => {
+    try {
+    let res = await fetch(
+        "https://www.themealdb.com/api/json/v1/1/filter.php?c=Beef"
+    );
+    final = await res.json();
+        let arr  = final.meals;
+    // console.log(final);
```
Comment (yaqootturman) [inline js/typeFood.js]: delete

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: js/typeFood.js (hunk 1 above)
Comment (muhannaAtalla20) [reply js/typeFood.js]: done

File: js/typeFood.js (hunk 2 above)
Comment (muhannaAtalla20) [reply js/typeFood.js]: done

**Review by yaqootturman — APPROVED** [review]

**Review by IsraaHamdi — APPROVED** [review]

**Review by farahalzard — APPROVED** [review]

**Pull request merged**